### PR TITLE
location updated to available true on insert

### PR DESCRIPTION
**Pull request by SanthoshGugan**
location update
update api for partner, user , customer

### CODE REVIEW

File: controllers/CustomerController.js [hunk 1]
```diff
@@ -129,5 +130,58 @@ router.post(`/addCustomer`, async (req, res, next) => {
...
+    try {
+
+        const {
+            id,
+            name,
+            location,
+            type
+        } = req.body;
+
+        const customer_id = req.    params.customerId;
```
Comment (premkumar91) [inline controllers/CustomerController.js:171]: typo between . and space, won't it break?

File: controllers/PartnerController.js [hunk 2]
```diff
@@ -80,6 +80,26 @@ router.post(`/addPartner`, async (req, res, next) => {
...
+router.put(`/`, async (req, res, next) => {
+    try {
+        const {
+            id,
+            name,
+            phone_number,
+            metadata
+        } = req.body;
+
+        const updatedPartner = await partnerService.updatePartner({
```
Comment (premkumar91) [inline controllers/PartnerController.js:92]: why update customer handles id from params and here it is from body? Any specific reason?

**Review by premkumar91 — DISMISSED** [review]

File: controllers/PartnerController.js (hunk 2 above)
Comment (SanthoshGugan) [reply controllers/PartnerController.js:92]: no specific reason, we will have to  make uniformity later I guess.

File: controllers/CustomerController.js (hunk 1 above)
Comment (SanthoshGugan) [reply controllers/CustomerController.js:171]: its typo buddy, updated

**Pull request merged**